hack/jenkins/e2e.sh

```diff
@@ -555,6 +555,8 @@ case ${JOB_NAME} in
     NUM_NODES="100"
     # Reduce logs verbosity
     TEST_CLUSTER_LOG_LEVEL="--v=2"
+    # TODO: Remove when we figure out the reason for ocassional failures #19048
+    KUBELET_TEST_LOG_LEVEL="--v=4"
```

Comment:
Do you mean isn't it too verbose? We can run it few times and get some failure data. I don't want to keep it this way for long.